internal/driver/webui_test.go

```diff
@@ -249,3 +249,49 @@ func TestIsLocalHost(t *testing.T) {
 		}
 	}
 }
+
+func TestFlameGraph(t *testing.T) {
+	prof := makeFakeProfile()
+	ui := &webInterface{
+		prof:    prof,
+		options: &plugin.Options{Obj: fakeObjTool{}},
+		help:    make(map[string]string),
+	}
+
+	// Start test server.
+	server := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			ui.flamegraph(w, r)
+		}))
+	defer server.Close()
+
+	type testCase struct {
+		path string
+		want []string
+	}
+	testcases := []testCase{
```

Comment:
Is there a difference between the handling of these test cases and the ones in TestWebInterface?  If not, maybe just move these cases into the existing function?  Will remove some duplicated code.

If you do that, will need to set needDot to false for these cases.